interactions/client/bot.pyi

```diff
@@ -67,23 +131,23 @@ class Client:
         options: Optional[List[Option]] = MISSING,
         default_permission: Optional[bool] = MISSING,
         name_localizations: Optional[Dict[Union[str, Locale], str]] = MISSING,
-        description_localizations: Optional[Dict[Union[str, Locale], str]]  = MISSING,
+        description_localizations: Optional[Dict[Union[str, Locale], str]] = MISSING,
```

Comment:
```suggestion
```

Why are you including this in a feature PR? Please revert.

Reply:
It was pre-commit, as you can see, it was removing a single unnecessary space.